conjur-api/Policy.cs

```diff
@@ -0,0 +1,44 @@
+﻿// <copyright file="Policy.cs" company="Conjur Inc.">
```

Comment:
Filed an issue (#23) to fix the copyright notices.

Reply:
Agree, will be done in a different PR for all files